samples/subsys/logging/logger/src/main.c

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <string.h>
+#include <misc/printk.h>
+
+#include <logging/log_backend_console.h>
+#include <logging/log_ctrl.h>
+#include <soc.h>
+#include "sample_instance.h"
+#include "sample_module.h"
+
+#define LOG_MODULE_NAME main
+#include <logging/log.h>
+LOG_MODULE_REGISTER();
+
+extern void sample_module_func(void);
+
+#define INST1_NAME STRINGIFY(SAMPLE_INSTANCE_NAME.inst1)
+SAMPLE_INSTANCE_DEFINE(inst1);
+
+#define INST2_NAME STRINGIFY(SAMPLE_INSTANCE_NAME.inst2)
+SAMPLE_INSTANCE_DEFINE(inst2);
+
+static u32_t timestamp_get()
+{
+#ifdef CONFIG_SOC_FAMILY_NRF
+	return NRF_RTC1->COUNTER;
```

Comment:
why? k_cycle_get_32() works on Nordic devices as well

Reply:
It does but it is very slow (see _timer_cycle_get_32 in nrf_rtc_timer.c). It temporary, wanted to use it to achieve expected performance. Hopefully, nordic implementation of cycle_get will get improved.